src/GraphQL/Subscription/SubscriptionExecutionResult.cs

```diff
@@ -9,12 +11,18 @@ public class SubscriptionExecutionResult : ExecutionResult
         /// </summary>
         public IDictionary<string, IObservable<ExecutionResult>>? Streams { get; set; }
 
-        /// <inheritdoc cref="ExecutionResult()"/>
+        /// <inheritdoc/>
         public SubscriptionExecutionResult()
         {
         }
 
-        /// <inheritdoc cref="ExecutionResult(ExecutionResult)"/>
+        /// <inheritdoc/>
+        internal SubscriptionExecutionResult(ExecutionContext context)
```

Comment:
You wanted to remove this class didn't you?

Reply:
Yes; that is listed above as to be done in a separate PR.